nova/virt/vmwareapi/vmops.py

```diff
@@ -2438,26 +2438,32 @@ def poll_rebooting_instances(self, timeout, instances):
 
     def get_info(self, instance):
         """Return data about the VM instance."""
-        lst_properties = ["runtime.powerState"]
+        powerstate_property = 'runtime.powerState'
```

Comment:
It doesn't make sense to have it as a list for the function as it stands. Because 2 out of 3 uses of this use it as string. Defining it a list, we would have to change those uses to check for/retrieve all the entries in the list.

If we pull it into a global constant, then I would not keep a local variable at all, so that does not benefit your use-case. Could you point me to the problematic could or give me more details, so we can derive a better solution?